media/server/gstplayer/source/GstGenericPlayer.cpp

```diff
@@ -408,6 +408,14 @@ GstElement *GstGenericPlayer::getSink(const MediaSourceType &mediaSourceType)
     }
     else
     {
+        if (m_context.pipeline == nullptr)
```

Comment:
These logs needed? Phil is doing some refactoring of getSink so I would leave as is for now.

Reply:
Yeh these logs aren't needed at all